File: app/src/main/java/com/pirateman/exercisemanager/exercise/ExerciseFragment.java
package com.pirateman.exercisemanager.exercise;

import android.databinding.DataBindingUtil;
import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.pirateman.exercisemanager.R;
import com.pirateman.exercisemanager.databinding.ExerciseItemBinding;


public class ExerciseFragment extends Fragment
{
    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState)
    {
        ExerciseItemBinding binding = DataBindingUtil.inflate(inflater, R.layout.recycler_view_exercise_item, container, false);

        Exercise exercise = new Exercise();
        exercise.setName("Chest Press");
        exercise.setMuscleGroup("Chest");
        exercise.setMethod("Barbells");

        binding.setExerciseRecord(exercise);

        return binding.getRoot();
    }
}

File: app/src/main/java/com/pirateman/exercisemanager/exercise/ExerciseFragment.kt
package com.pirateman.exercisemanager.exercise

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Toast
import androidx.fragment.app.Fragment
import androidx.lifecycle.Observer
import com.pirateman.exercisemanager.R
import org.koin.android.ext.android.inject
import org.koin.java.KoinJavaComponent.inject

class ExerciseFragment : Fragment() {

    val exerciseViewModel: ExerciseViewModel by inject()

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {

        exerciseViewModel.getAllExercises().observe(this.viewLifecycleOwner, Observer {

            for (i in it){
                Toast.makeText(this.context, i.name, Toast.LENGTH_SHORT).show()
            }
        })
        return inflater.inflate(R.layout.recycler_view_exercise_item, container, false)
    }
}